libs/features/pages/contact-page/src/lib/components/contact-page-form-section/contact-page-form-section.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+import { SliderButtonData } from '@elewa-website/models/schema/ui/buttons';
+
+@Component({
+  selector: 'elewa-website-contact-page-form-section',
+  templateUrl: './contact-page-form-section.component.html',
+  styleUrls: ['./contact-page-form-section.component.scss'],
+})
+export class ContactPageFormSectionComponent {
+  contactForm: FormGroup;
+
+  constructor(private fb: FormBuilder) {
+    this.contactForm = this.fb.group({
+      name: ['', Validators.required],
+      company: [''],
+      email: ['', [Validators.required, Validators.email]],
+      reason: ['general', Validators.required],
+      message: ['', Validators.required],
+    });
+  }
+  onSubmit() {
+    /**Form submission functionality */
+  }
+
+  sendButtonData: SliderButtonData = {
+    text: "Send message"
+  }
```

Comment:
For keeping the component lean and ensuring readability and maintainability, this can be moved to a Typescript file in the data module then referenced from there